chat-plugins/dice-commands.js

```diff
@@ -1,185 +1,167 @@
-/* Dice commands chat-plugin
- * by jd
-*/
-var tax = 0.10; // 10%
+/*Dice Plugin by SilverTactic (Silveee)*/
 
-function handleWinnings (bet) {
-	return bet - Math.round(bet * tax);
+'use strict';
+
+if (!global.dicegames) {
+	global.dicegames = {};
+	global.dicegames.count = 0;
+}
+let dicegames = global.dicegames;
```

Comment:
You're right, it's probably not necessary. Should I make it <code>Rooms('gamechamber').dicegames</code> instead?